main.go

```diff
@@ -333,22 +326,23 @@ func (a *ATM) theFirstEntering() {
 		fmt.Scan(&cardNum)
 		fmt.Println("Идёт считывание карты")
 
-		if !a.checkClientForExistance(cardNum) {
-			a.createNewClient(cardNum)
-		} else {
-			if !a.checkTheFirstEnteringPincode() {
+		if a.doesClientExist(cardNum) {
+			if !a.isPinСodeCorrect() {
 				return
 			}
+			a.identifyTheСlient(cardNum)
+		} else {
+			a.createNewClient(cardNum)
```

Comment:
client, exists := getClient(cardNum)
if exists {
       if ! a.isPinCodeCorrect() {
		return incorrectPinCode
	}
} else {
	client = a.createNewClient(cardNum)
}

Reply:
func (a *ATM) getClient(cardNum int) (Client, bool) {
	for _, v := range a.Clients {
		if v.CardNumber == cardNum {
			return v, true
		}
	}
	return Client{}, false
}